lib/resources/forms.js

```diff
@@ -325,37 +325,39 @@ module.exports = (service, endpoint, anonymousEndpoint) => {
   };
 
   formResource('/projects/:projectId/forms/:xmlFormId', (Forms, params, withXml = false, options = QueryOptions.none) =>
-    Forms.getByProjectAndXmlFormId(params.projectId, params.xmlFormId, withXml, null, options)
-      .then(getOrNotFound));
+    Forms.getByProjectAndXmlFormId(params.projectId, params.xmlFormId, Form.AnyVersion, Form.withXml(withXml), options)
+      .then(getOrNotFound)
+      .then(ensureDef));
   formResource('/projects/:projectId/forms/:xmlFormId/versions/:version', (Forms, params, withXml = false, options = QueryOptions.none) =>
-    Forms.getByProjectAndXmlFormId(params.projectId, params.xmlFormId, withXml, params.version, options)
+    Forms.getByProjectAndXmlFormId(params.projectId, params.xmlFormId, params.version, Form.withXml(withXml), options)
       .then(getOrNotFound)
       .then(ensureDef));
 
   formResource('/projects/:projectId/forms/:xmlFormId/draft', (Forms, params, withXml = false, options = QueryOptions.none) =>
-    Forms.getByProjectAndXmlFormId(params.projectId, params.xmlFormId, withXml, Form.DraftVersion, options)
+    Forms.getByProjectAndXmlFormId(params.projectId, params.xmlFormId, Form.DraftVersion, Form.withXml(withXml), options)
       .then(getOrNotFound)
       .then(ensureDef));
 
   service.get('/form-links/:enketoId/form', endpoint(({ Forms }, { params, auth }) =>
-    Forms.getByEnketoId(params.enketoId, false, null, QueryOptions.none)
+    Forms.getByEnketoId(params.enketoId)
```

Comment:
Confirm if this API is mentioned in PR description, or if it should be a separate PR.